src/Components/UploadModal/UploadModal.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import "../UploadModal/UploadModal.scss";
+
+const UploadModal = ({ visible, toggle }) =>
+  visible
+    ? ReactDOM.createPortal(
```

Comment:
I don't think this.props would work since this is a functional component.
I found ReactDOM.createPortal is a easier way to render children onto the app while also being easier to manage in the future (in terms of scalability).